.gitpack/data/asus-fan-control.service

```diff
@@ -10,6 +10,7 @@ After=hibernate.target suspend-then-hibernate.target
 
 [Service]
 Type=oneshot
+ExecStartPre=-/sbin/modprobe acpi_call
```

Comment:
Is there any reason not to remove the `-`? As long as know, modprobe should return success in case a given kernel module is loaded after running it. It works for me even without root permissions in case a given model has been already loaded. Why should we voluntarily expose asus-fan-control to the state where acpi_call is not loaded and we know about it (I hope I have understood the `-` correctly). It should end up with an error anyways, but I think the systemd error would be more clear here.

Reply:
Yeah, you're understanding it correctly, if the `ExecStartPre` fails, `Exec` isn't even attempted. Well, as you said, if it failed, asus-fan-control's gonna fail as well. There might be some edge cages though, I made up one but after some thinking found out it's invalid. But there might be valid ones we're missing! Either way, it does no harm, and `asus-fan-control` is ready to that situation and exits with an according message.

Reply:
I still see no real point of letting `asus-fan-control` run after a `modprobe` fail. What about to remove `-` now and we can always add it back once we figure out an edge case?

Just trying to keep it simple... :rocket:

Reply:
Done :rocket: